src/AuthorizationCodeHandler.ts

```diff
@@ -54,9 +64,14 @@ class AuthorizationCodeHandler implements LambdaInterface {
 						return new Response(HttpCodesEnum.SERVER_ERROR, "An error has occurred");
 					}
 				}
+				logger.warn("Method not implemented", { messageCode: MessageCodes.METHOD_NOT_IMPLEMENTED });
 				return new Response(HttpCodesEnum.NOT_FOUND, "");
 
 			default:
+				logger.error("Requested resource does not exist", {
+					resource: event.resource,
+					messageCode: MessageCodes.RESOURCE_NOT_FOUND,
+				});
 				throw new AppError(HttpCodesEnum.NOT_FOUND, "Requested resource does not exist" + { resource: event.resource });
```

Comment:
just to check - where is this exception caught?

Reply:
modified to return 404